01-Introducing-Molecular-Dynamics/02b-Running-Rigid-Body-Molecular-Dynamics-System.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p><code>state</code>  is no longer a logger category. Well it shouldn't be... I forgot to remove it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>